### PR TITLE
[GAL-385] Add Activity View (User Feed) to User Gallery page

### PR DESCRIPTION
**Pull request by jakzaizzat**
**Changes**
- Added user activity feed
- Access it directly through https://gallery.so/username/activity

**Demo**

**Mobile**

https://user-images.githubusercontent.com/4480258/196184395-23dc714a-9b7a-4a6e-bceb-d69b074c44bc.mp4

**Desktop**

https://user-images.githubusercontent.com/4480258/196184753-5c307752-2a8a-4e99-af8c-92b49a87e26f.mp4

### CODE REVIEW

**Comment by tbezman** [comment]
One slightly annoying thing is the click targets for the feed events seem to be wider than the actual content 
image

File: src/scenes/UserActivityPage/UserActivityLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    return () => {
+      // [GAL-302] figure out a cleaner way to do this. prevent dismount of follow icon
+      // if we're transitioning in between pages on the same user. otherwise there's a
+      // race condition between this page trying to dismount the follow icon vs. the next
+      // page trying to mount it again
+      if (window.location.href.includes(user.username ?? '')) {
+        return;
+      }
+      setCustomNavLeftContent(null);
```
Comment (tbezman) [inline src/scenes/UserActivityPage/UserActivityLayout.tsx:65]: good callout.

**Comment by jakzaizzat** [comment]
> One slightly annoying thing is the click targets for the feed events seem to be wider than the actual content image

Good catch @tbezman ! I've update the max width of the feed (similar to home feed)

**Review by Robinnnnn — COMMENTED** [review]
@jakzaizzat this looks so amazing!!!

for some reason when i reduce the screen width to mobile, the feed doesn't appear anymore? any reason why 🤔 

image

**Pull request merged**